src/app/containers/ImageWithPlaceholder/__snapshots__/index.test.jsx.snap

```diff
@@ -15,66 +15,29 @@ exports[`ImageWithPlaceholder should not provide non-js fallback 1`] = `
-}
-
-.c1 {
-  display: block;
-  width: 100%;
-  visibility: visible;
-  -webkit-animation: eMLfYp 0.2s linear;
-  animation: eMLfYp 0.2s linear;
-  -webkit-transition: visibility 0.2s linear;
-  transition: visibility 0.2s linear;
-}
-
-<div
-  className="c0"
->
-  <div
-    className="lazyload-placeholder"
-    style={
-      Object {
-        "height": undefined,
-      }
-    }
-  />
-  <noscript>
-    <img
-      alt="Pauline Clayton"
-      className="c1"
-      height={null}
-      sizes={null}
-      src="https://ichef.bbci.co.uk/news/640/cpsprodpb/E7DB/production/_101655395_paulineclayton.jpg"
-      srcSet="https://ichef.bbci.co.uk/news/640/cpsprodpb/E7DB/production/_101655395_paulineclayton.jpg 640w"
-      width={640}
-    />
-  </noscript>
-</div>
+<WrappedImageWithPlaceholder
```

Comment:
This is just testing the component in fixture data is called :( https://github.com/bbc/simorgh/blob/remove-test-helpers-7/src/app/containers/ImageWithPlaceholder/fixtureData.jsx#L11

Howcome the shallow render? I dont think it can be used here